### DISCUSSION TITLE
Do not eslint/prettier and change all source files without permission! (grafana-toolkit build)

### DISCUSSION THREAD

**Discussion by cyraid** [grafana/grafana#44927] [General]
**What happened**:
All my source code style got wiped by your eslint/prettier rules on a build of a plugin.

**What you expected to happen**:
Do NOT change source files by default.

**How to reproduce it (as minimally and precisely as possible)**:
Go against the grain and do ' quotes instead of " (just as a start), then build.

**Anything else we need to know?**:
This caused me quite a bit of grief, and really should never change source files by a build. :) Even a lint. ALWAYS leave the source alone unless instructed.

**Environment**:
- Grafana version: 8
- Data source type & version: 
- OS Grafana is installed on: Ubuntu
- User OS & Browser: Chrome
- Grafana plugins: custom
- Others:

**Comment by cyraid** [comment]
@matschaffer @kogent @greut @svenstaro Does nobody have input on this? Can someone at least tell me how to disable this completely? I'm getting tired of it *changing* my source code when I do grafana-toolkit plugin:dev

**Reply by svenstaro** [reply]
I have nothing to do with this project.

**Reply by cyraid** [reply]
Sorry Sven, not sure why your name out of the few popped up when I typed @ .. assumed it would've autocomplete with relevant members.

**Comment by torkelo** [comment]
If you place a .prettierrc.js  file in the root of your repo you can set your own prettier config

**Reply by cyraid** [reply]
What exactly do I put in .prettierrc.js to disable it completely?

**Reply by RedShift1** [reply]
Try `grafana-toolkit plugin:build --skipLint`

Edit: found the command args here: https://www.npmjs.com/package/@grafana/toolkit (it won't show up with --help)

**Reply by cyraid** [reply]
> Try `grafana-toolkit plugin:build --skipLint`
> 
> Edit: found the command args here: https://www.npmjs.com/package/@grafana/toolkit (it won't show up with --help)

I was using that for build, but there's no skip lint for any other mode. But I don't work with grafana anymore for the moment.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
